### DISCUSSION TITLE
Raspberry Placement

### DISCUSSION THREAD

**Discussion by WillE726** [tesla-android/issue-tracker#249] [Q&A]
Where do you all place the raspberry pi in the Tesla for a good connection?

**Comment by mikegapinski** [comment]
I use the center console arm rest. For older cars with the wifi antenna on the outside place it as close to the right a pillar as possible

**Reply by WillE726** [reply]
Ok yeah I have 2019 model x and I am trying to find a good place for it.  Thanks

**Reply by Cooker44** [reply]
Hi Mike
Can a line be drawn under ‘older cars’? Which year/model, or is there an online resource to help…can’t find it in the manual. In any event, I’m 2017 so moving to the right A pillar! (Assuming it’s right even on right hand drive vehicles!)

**Reply by mikegapinski** [reply]
Old cars(antenna in the mirror)

All legacy s/x
Model 3 with the old center console.

The new version scheduled to release soon works better on MCU2 cars btw ;)

**Reply by Cooker44** [reply]
Wrt. LHD/RHD…legacy S&X

Antenna. The vehicle's WiFi antenna is in the right side mirror housing on both LHD and RHD vehicles. The best signal strength occurs with a minimum of walls and objects between the antenna and your WiFi router.

**Reply by mikegapinski** [reply]
@Cooker44 I've heard there is a connector for it somewhere in the A pillar and you can plug another antena to it. Have not tried yet but it's  a good alternative to taking the MCU out.

**Comment by itzik2sh** [comment]
I bought  a drawer in amazon to fit the center console, drilled 2 holes in it and positioned the RPI in it with an ON-OFF push button glued with a double sided 3M tape. Looks like this
![WhatsApp Image 2023-04-22 at 20 38 26](https://user-images.githubusercontent.com/1967268/233835216-1a39c648-6ab9-4ef8-8cfc-7aa0524d0791.jpeg)

**Comment by Cooker44** [comment]
Thanks, Mike, that’s helpful.
